src/css/main.css

```diff
@@ -572,3 +585,31 @@ dl.glossary {display: block;}
 		padding: 0.5rem 0 1rem 1rem;
 		border-left: 1px solid var(--main-color);
 	}
+
+/* --------------------- WALLED GARDENS PAGE ------------------ */
+
+div.article-data {
+	margin: -3rem 0 3rem;
+}
+
+p.baseline {
+	font-size: 120%;
+	font-variation-settings: var(--fvs-bold);
+	line-height: 1.3;
+	margin-bottom: 0.5em;
+}
+
+.p-author a {font-size: 80%;}
+
+div.toc ul {margin-left: 2.5rem;}
+div.toc li {font-size: 0.9em;}
+
+p.download {
+	background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' width='24' height='24'%3E%3Cpath fill='none' d='M0 0h24v24H0z'/%3E%3Cpath d='M16 2l5 5v14.008a.993.993 0 0 1-.993.992H3.993A1 1 0 0 1 3 21.008V2.992C3 2.444 3.445 2 3.993 2H16zm-3 10V8h-2v4H8l4 4 4-4h-3z' style='fill:%23CD4C1D' /%3E%3C/svg%3E");
```

Comment:
If this image might be reused, would it be possible to tuck it into a css variable?  A named variable would give some context to what we're loading as well.

Reply:
Yes, no problem. Not sure it will be used a lot though (or modified)